docs/tweaks/ddata.js

```diff
@@ -0,0 +1,34 @@
+var util = require('util')
+var arrayify = require('array-back')
+exports.anchorName = anchorName
+
+function anchorName(options) {
+    var methodSign = options.data.methodSign ? options.data.methodSign.replace(/[^\w,]/g, '-').replace(/^-+|-+$/g, '').replace(/-+/g, '').replace(/,/g, '-').toLowerCase() : '';
```

Comment:
Noted in the sonar

![imagen](https://github.com/aragon/sdk/assets/36479864/8260595a-3442-47c0-ac5c-eca37dfb8edd)